vulture.py

```diff
@@ -165,8 +167,14 @@ def _get_modules(self, paths, toplevel=True):
                 sys.exit('Error: %s could not be found.' % path)
         return modules
 
+    def _get_stdlib_whitelist_file(self):
```

Comment:
This should be a function, not a method, since it doesn't need access to the Vulture object.